challenges/01 Goldilocks' Bear Necessities/app.js

```diff
@@ -1 +1,58 @@
-// Do awesome stuff...
+const fs = require('fs');
+
+//  Einlesen der Daten aus dem 'inputs.txt'-File.
+//  Wenn keine Daten im File gefunden werden,
+//  wird eine Fehlernachricht ausgegeben
+//  und der Prozess beendet.
+const rawFileInput = fs.readFileSync('inputs.txt');
+if(!rawFileInput){
+    console.log('Keine Daten gefunden');
+    process.exit(1);
+}
+
+//  Umwandeln des Inputs in ein Array aus Wertepaaren.
+const stringFileInput = rawFileInput.toString();
+const valuePairs = stringFileInput.split(/\n/);
+
+
+//  Abspeichern von Goldilocks Werten und Entfernen des ersten Wertepaares,
+//  da es nur für Goldilocks Werte wichtig ist.
+const goldisValues = valuePairs.shift()
+const goldilocksWeight = parseInt(goldisValues.split(' ')[0]);
+const goldilocksTemp = parseInt(goldisValues.split(' ')[1]);
+
+//  Leeres Array für passende Wertepaare initialisieren.
+const suitablePairs = [];
+
+//  Wertepaare-Array durchlaufen und suitable-Funktion anwenden.
+// Wenn diese 'true' zurückgibt, wird das aktuelle Wertepaar gespeichert.
```

Comment:
Das ist mehr ein "Übersetzen von Code in Sprache". Hier vielleicht mehr darauf eingehen, was in _suitablePairs_ gespeichert wird.